README.md

```diff
@@ -20,18 +20,13 @@
 
 * [Research](RESEARCH.md)
 
-## Links
-
-* [Fission Reactor: Dialog First Look](https://fission.codes/blog/fission-reactor-dialog-first-look/)
```

Comment:
Now that we have an entire org, we should move these resources to a common place. TODO: move this specific one to the `.github/profile`